decidim-core/app/views/layouts/decidim/participatory_process.html.erb

```diff
@@ -1,46 +1,55 @@
 <%= render "layouts/decidim/application" do %>
-<main class="wrapper">
-  <div class="process-header process-header--home row column">
-    <div class="row column process-header__main"
-      style="background-image:url(<%= current_participatory_process.banner_image.url %>);">
-      <div class="process-header__container row">
-        <div class="columns mediumlarge-9 process-header__info">
-          <h1 class="text-highlight heading1">
-            <%= translated_attribute(current_participatory_process.title) %>
-          </h1>
-          <h2 class="text-highlight heading4">
-            <span class="process-header__hashtag">
-              <%= current_participatory_process.hashtag %>
-            </span>
-            <%= translated_attribute(current_participatory_process.subtitle) %>
-          </h2>
+  <div class="wrapper">
```

Comment:
Hi @htmlboy ! Related to @oriolgual's comment I'm seeing in some templates a `main` tag with class `wrapper` and in other ones a `div` tag with class `wrapper`:
See:
- https://github.com/AjuntamentdeBarcelona/decidim-design/blob/master/source/proposal-view.html.erb
- https://github.com/AjuntamentdeBarcelona/decidim-design/blob/master/source/new-proposal.html.erb

Which one is correct? 😄 

Also... I'm not sure about this one but in the generated html I'm seeing the main content inside a `div` with class `footer-separator`. Is it correct?

![image](https://cloud.githubusercontent.com/assets/106021/21219979/31afc0a6-c2b7-11e6-949d-c4de338d7ebe.png)